perf-tests/clusterloader2/pkg/measurement/util/object_store.go

```diff
@@ -80,9 +85,17 @@ func (s *ObjectStore) Stop() {
 // PodStore is a convenient wrapper around cache.Store.
 type PodStore struct {
 	*ObjectStore
-	listener int
+	listener      int
+	podListerFunc func(labelKey string, labelValue string) ([]*v1.Pod, error)
 }
 
+const (
+	labelNameKeyIndex = "label.name"
```

Comment:
do we need both indexers ? just group should be good for the latency pods which we only care about, right ?

Reply:
In local test, somehow there is case that FilterPods get selector with group=saturation only. It falls into checking each label case, which could still take a lot of time.

Reply:
It is used for the periodically print out "SchedulingThroughput: # pods scheduled"

Reply:
if we can only have one index, the code change can be much simplified.

Reply:
> if we can only have one index, the code change can be much simplified.

The code change is mainly for efficiency. W/o the new group index, the label comparision will need to be done for every single pod in saturation pod creation step. It can also be very CPU intensive.